### PR TITLE
Update SMC-ABC_Lotka-Volterra_example.ipynb

### PR DESCRIPTION
**Pull request by PRKramer**
#588  Added the time array to the plots to correctly represent the independent variable in 2 plots

+ [ ] Notebook follows style guide https://docs.pymc.io/en/latest/contributing/jupyter_style.html
+ [ ] PR description contains a link to the relevant issue:
  * a tracker one for existing notebooks (tracker issues have the "tracker id" label)
  * or a proposal one for new notebooks
+ [ ] Check the notebook is not excluded from any pre-commit check: https://github.com/pymc-devs/pymc-examples/blob/main/.pre-commit-config.yaml

### Helpful links
* https://github.com/pymc-devs/pymc-examples/blob/main/CONTRIBUTING.md

----
:books: Documentation preview :books:: https://pymc-examples--590.org.readthedocs.build/en/590/

### CODE REVIEW

**Comment by OriolAbril** [comment]
Thanks for the PR. You should also rerun the notebook and then run pre-commit

**Comment by PRKramer** [comment]
Sorry I am really naive about python and GitHub.  I just noticed the error in the notebook when I was adopting the material for use in a math class.  I can't even figure outhow to run python on my modified notebook, nor how to download the notebook onto my computer.  I just modified the syntax of the plot commands in a way that worked when I did it on a private notebook.  I don't expect you to give me a tutorial on pull requests, nor do I want to read a massive amount of documentation.  I just wanted to point out a small error in the example notebook which should be very simple to fix for someone who understands the GitHub process better than I do...I'm happy to abandon the issue if it's more bother than it's worth.

**Comment by PRKramer** [comment]
Wait..one of my students in my class figured out how to overcome the check failure...working on it!

**Comment by MacroLens** [comment]
@OriolAbril reran the notebook and ran pre-commit on changes.
Let me know if there is anything else you need.

**Comment by PRKramer** [comment]
I was flying blind because I didn't see how to actually run the change.  Dylan reminded me about cloning (yes, it's been a few years since my last GitHub encounter with some research students, which sent me fleeing).  So I'll give it a closer look when I have a bit of time to walk through the mechanics.

**Review by OriolAbril — COMMENTED** [review]
I added a couple comments. They are on the myst file, but the changes should be made on the notebook, it's pre-commit that syncs the two up again. That should fix the wrong looking plot from the reviewNB comment.

Side note: we are having a docathon starting in 5 minutes (will go on for 3 hours). In case it might be of interest, more info on [pymc's discourse](https://discourse.pymc.io/t/pymc-docathon-elevate-open-source-documentation/13248)

**Comment by MacroLens** [comment]
<p>You would be correct. I'm fixing it now, the time scale wasn't applied to the plot on line 6.</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/590/discussion/#comment-1815896420'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by MacroLens** [comment]
<p>Looks better now.</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/590/discussion/#comment-1815896420'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: examples/samplers/SMC-ABC_Lotka-Volterra_example.myst.md [hunk 1]
```diff
@@ -198,8 +198,8 @@ az.plot_posterior(idata_lv);
 # plot results
 _, ax = plt.subplots(figsize=(14, 6))
 posterior = idata_lv.posterior.stack(samples=("draw", "chain"))
-ax.plot(observed[:, 0], "o", label="prey", c="C0", mec="k")
-ax.plot(observed[:, 1], "o", label="predator", c="C1", mec="k")
+ax.plot(t, observed[:, 0], "o", label="prey", c="C0", mec="k")
+ax.plot(t, observed[:, 1], "o", label="predator", c="C1", mec="k")
 ax.plot(competition_model(None, posterior["a"].mean(), posterior["b"].mean()), linewidth=3)
 for i in np.random.randint(0, size, 75):
     sim = competition_model(None, posterior["a"][i], posterior["b"][i])
```
[the remark this answers is not in the document]
Comment (MacroLens) [reply examples/samplers/SMC-ABC_Lotka-Volterra_example.myst.md:205]: Oops I misinterpreted what you meant. I fixed this in [f171270](https://github.com/pymc-devs/pymc-examples/pull/590/commits/f1712708fac720d6ad8e231a763ad6746e160af3).

**Review by twiecki — APPROVED** [review]

**Comment by twiecki** [comment]
Thanks!

**Comment by Fxinyu** [comment]
Hi, I have encountered this problem. Do 
Screenshot 2023-11-23 102841
you have any insights on this situation? I would appreciate that you could reply to me.

**Comment by Fxinyu** [comment]
Hi, I have encountered this problem. Do you have any insights on this situation? I would appreciate that you could reply to me.
Screenshot 2023-11-23 102841

**Comment by twiecki** [comment]
Are other models sampling fine?

**Pull request merged**